starkup.sh

```diff
@@ -225,60 +226,75 @@ is_asdf_legacy() {
 }
 
 install_asdf_interactively() {
-  _profile=""
-  _pref_shell=""
-  _completion_message=""
-  _asdf_path="$HOME/.asdf"
-
-  case ${SHELL:-""} in
-  */zsh)
-    _profile=$HOME/.zshrc
-    _pref_shell=zsh
-    _completion_message="Run 'source ${_profile}'"
-    ;;
-  */bash)
-    if [ "$(uname)" = "Darwin" ]; then
-      _profile=$HOME/.bash_profile
-    else
-      _profile=$HOME/.bashrc
+  say "asdf-vm is required. It can be installed via package managers, including Homebrew and Pacman.\nFor more information, visit ${ASDF_INSTALL_DOCS}.\nAlternatively, an asdf binary can be installed by starkup.\nDo you want to install it now? (y/N):"
```

Comment:
nitpick: the pacman part is very misleading (and I know asdf writes this on their help page, but this is simply not true). asdf is available through AUR (Arch User Repository), not through pacman (official distro repository). Let's change that :+1:

Reply:
Hmm, not sure how to include this in a compact way as to not confuse the user 🤔 

```suggestion
  say "asdf-vm is required. \nIt can be installed via package managers, including Homebrew and Pacman (available through AUR).\nFor more information, visit ${ASDF_INSTALL_DOCS}.\nAlternatively, an asdf binary can be installed by starkup.\nDo you want to install it now? (y/N):"
```

Wdyt?

Reply:
I'd say something like: "can be installed using package managers, including Homebrew, or any AUR helper."

But frankly, I'm not sure I'd even mention AUR. Arch users are **chads** and they will already know asdf is there (everything is there XD). If they managed to install Arch, they will surely manage to install asdf :+1:

Reply:
yup, either one of these suggestions is great :+1: Just go with wichever suits you best